src/develop/tutorial/smart-digital-assets/issue-ft.md

```diff
@@ -1,15 +1,508 @@
-  Also please make sure to consult our rules and rewards policy via [this link](https://docs.everscale.network/contribute/hot-streams/documentations).  
-  Feel free to join [Everscale Documentation Development Telegram chat](https://t.me/+C2IpQXWZtCwxYzEy) and [Everscale Developers Onboarding Telegram chat](https://t.me/+Vca1Gs6uPzIyNWVi)!
+## Prerequisites
+
+Before we begin, make sure that you already have the following installed:
+
+- [NodeJs](https://nodejs.org/) - version 16 or later
+- [Docker](https://www.docker.com/) - version 19 or later
+
+## Getting Started
+
+To begin with, let us recall the architecture of the [TIP-3 token](../../../standard/TIP-3/core-description.md):
+
+- The information about the token is stored in the **Token root** contract.
+- Each holder has its own instance of **Token Wallet**. This contract is used to store tokens.  
+It also can send tokens to any smart contract or receive them from another Token Wallet.
+
+As an example, we will use the [TIP-3 token repository](https://github.com/broxus/tip3) by Broxus.  
+However, we will rewrite the scripts in this repository in Typescript for the new version of Locklift.
+
+### Creating a project
+
+The first thing we need to do is to install [Locklift](https://www.notion.so/tools/locklift.md) and create a new project.
+Locklift is a development environment designed to help writing Everscale smart contracts.
+
+Start by creating a folder.
+We'll call it `tip-3`, but you can choose any other name you like.
+
+```shell
+mkdir tip-3
+cd tip-3
+```
+
+Afterwards, create an NPM project.
+
+```shell
+npm init
+```
+
+Now, we need to install the Locklift locally. If you have a globally installed Locklift, you can skip this step.
+
+```shell
+npm install --save-dev locklift
+```
+
+To install Locklift globally, you can use the following command:
+
+```shell
+npm install -g locklift
+```
+
+After that, initialize the project with this command:
+
+```shell
+npx locklift init -f
+```
+
+As a result of the initialization, a new Locklift project will be created, filled with samples:
+
+```
+├── contracts
+│   └── Sample.sol
+├── locklift.config.ts
+├── scripts
+│   └── 1-deploy-sample.ts
+├── giverSettings
+|   └── index.ts
+└── test
+    └── sample-test.ts
+```
+
+### Locklift config settings
+
+Since we will be using a sandbox, it is nedded to remove `graphql` from `LOCAL_NETWORK_ENDPOINT` in `locklift.config.ts` :
+
+```ts
+const LOCAL_NETWORK_ENDPOINT = "<http://localhost/graphql>";
+// to
+const LOCAL_NETWORK_ENDPOINT = "<http://localhost/>;
+```
+
+### Add Sandbox
+
+Let's add a Sandbox script to `package.json`, with which we will launch a local node.
+
+```
+ "scripts": {
+    "test": "npx locklift test --network local",
+    "start-sandbox": "docker run -d -e USER_AGREEMENT=yes --rm --name local-node -p80:80 tonlabs/local-node:0.29.1",
+    "stop-sandbox": "docker kill local-node",
```

Comment:
docker stop is a more friendly command

Reply:
use `everdev se start` and  `everdev se stop`

Reply:
https://github.com/everscale-org/docs/blob/9d8035652d0f049c8184d4effc8f588e79d7123e/src/develop/tutorial/smart-digital-assets/issue-ft.md?plain=1#L97

Reply:
so mb `npx everdev se start` like `npx locklift test --network local` command